public/components/register_model/artifact.tsx

```diff
@@ -5,18 +5,18 @@
 
 import React, { useState } from 'react';
 import { EuiTitle, htmlIdGenerator, EuiSpacer, EuiText, EuiRadio, EuiLink } from '@elastic/eui';
-
 import { MAX_MODEL_FILE_SIZE, ModelFileUploader } from './artifact_file';
 import { ArtifactUrl } from './artifact_url';
 import { ONE_GB } from '../../../common/constant';
-
 export const ArtifactPanel = () => {
   const [selectedSource, setSelectedSource] = useState<'source_from_computer' | 'source_from_url'>(
     'source_from_computer'
   );
-
   return (
     <div>
+      <EuiTitle size="m">
+        <h1>File and version information</h1>
```

Comment:
Do we need to use Small and h2 for this title? Just find this font in the figma.
<img width="843" alt="image" src="https://user-images.githubusercontent.com/4034161/222359002-552ddbea-de09-47c8-b46f-568456d7bf75.png">